src/DIRAC/Core/Utilities/Os.py

```diff
@@ -150,3 +152,25 @@ def sourceEnv(timeout, cmdTuple, inputEnv=None):
 @deprecated("Will be removed in DIRAC 8.1", onlyOnce=True)
 def which(executable):
     return shutil.which(executable)
+
+
+def findImage(operating_system="alma9"):
+    """Finds the image for the current platform"""
+    from DIRAC.ConfigurationSystem.Client.Helpers.Operations import Operations
+
+    plat = DIRAC.gConfig.getValue("LocalSite/Platform", platform.machine())
+    if plat == "x86_64":
+        plat = "amd64"
+
+    CVMFS_locations = DIRAC.gConfig.getValue(
+        "LocalSite/CVMFS_locations", Operations().getValue("Pilot/CVMFS_locations", [])
+    )
+
+    rootImage = None
+
+    for candidate in CVMFS_locations:
+        rootImage = os.path.join(candidate, "containers", "os-base", f"{operating_system}-devel", "prod", plat)
```

Comment:
I think we should be less opinionated here and just specify a prefix that gets appended with the platform.

Reply:
For example?

Reply:
```bash
for candidate in imageLocations:
    os.path.join(candidate, plat)
```

Reply:
It's following Marco's naming scheme for the container images with `os-base/xxx-devel`

Reply:
I've asked the developers behind `/cvmfs/unpacked.cern.ch` to see what they plan to do for laying out multi-platform images on CVMFS.

Reply:
> I've asked the developers behind `/cvmfs/unpacked.cern.ch` to see what they plan to do for laying out multi-platform images on CVMFS.

Hi, do you have an answer on this? 

I would like to conclude on this issue/PR.

Reply:
It should definitely be documented somewhere, where the container images are looked for, probably best in the SingularityCE module docstring.

Reply:
Can we have some debug printouts of `f"looking at {rootImage}"`?

Reply:
> It should definitely be documented somewhere, where the container images are looked for, probably best in the SingularityCE module docstring.

I agree, I am waiting for the approval of the "RFC" to add the documentation.